pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,286 @@
+
+Keep an eye out for announcements and updates. While Bento AMP is in developer preview, the team will continue to make necessary changes to the API to improve features.
+
+
+## Enable Bento Experiment 
+
+Bento components are available experimentally and you must [enable experimental features](../learn/experimental.md).
+
+Enable the Bento experiment by including the script below:
+
+
+```
+<script>
+  (self.AMP = self.AMP || []).push(function (AMP) {
+    AMP.toggleExperiment('bento', true);
+  });
+</script>
+```
+
+## Import AMP runtime and Bento component script
+
+You must include the AMP runtime script and import the component script for each individual Bento component desired.
+
+```
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+
+<script async custom-element="amp-bento-component-name" src="https://cdn.ampproject.org/v0/amp-bento-component-name-1.0.js"></script>
+``` 
+
+[tip type="note"]
+To make Bento components load event faster, eliminating need for the AMP runtime script is a high priority for Bento AMP. Follow our progress on the [Bento roadmap](../../../community/roadmap.html?category=bento#status-updates).
+[/tip]
+
+Read each Bento component’s reference documentation for implementation details.
+
+## Layout and styling
+
+Bento components require a small CSS library (??kb) to appear properly without content shifts on load [content shifts](https://web.dev/cls/). Include the following script tag in your document or import it into your build process.
+
+```
+<style src="..."></style>
+```
+
+Include additional standard CSS to further define layouts and styles. Here are a few strategies on how you can layout Bento components using CSS:
+
+### Fixed layout
+
+```css
+.fixed-layout {
+    display: inline-block;
+    width: xpx;
+    height: xpx;
+}
+```
+
+### Container layout
+
+```css
+.container-layout {
+...
+}
+```
+
+### Responsive layout with a fixed aspect ratio
+
+```css
+.responsive-layout {
+    display: block;
+    width: 100%;
+    height: auto;
+    aspect-ratio: width / height;
+}
+```
+
+### Fill layout
+
+```css
+.fill-layout {
+...
+}
+```
+
+### Flex layout
+
+```css
+.flex-layout {
+    display: block;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 1;
+    max-height: 100%;
+    max-width: 100%;
+    min-height: 100%;
+    min-width: 100%;
+}
+```
+
+### Grid layout
+
+```css
+.grid-layout {
+...
+}
+```
+
+[tip type="note"]
+
+Bento components don't support [AMP's layout](https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-layout/). 
+
+[/tip]
+
+## Interacting with Bento components
+
+Bento components have individual APIs that grant access to component actions and events. Access to component APIs and invoking events use the following syntax:
+
+```
+await customElements.whenDefined('amp-bento-component');
```

Comment:
Do you think it's worth commenting that this only needs to be done once per all `amp-bento-component` elements in the document? Or is it self-obvious?

Reply:
Thanks for the catch! I've added some clarification - it's always worth clarifying anything that may be confusing.